sdk/src/androidTest/java/com/bugsnag/android/ErrorTest.java

```diff
@@ -314,28 +314,13 @@ public void testBugsnagExceptionName() throws Exception {
         assertEquals("Busgang", err.getExceptionName());
     }
 
-    @Test
-    public void testConfigContext() throws Exception {
-        String expected = "Junit test suite";
-        error.setContext(null);
-        config.setContext(expected);
```

Comment:
This test could be reworked so that `config.setContext` is called before the error is built.